README.md

```diff
@@ -1,13 +1,11 @@
-# Project Name
+<strong><h1>Guess who game</h1> 🤠	🤡</strong>
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The goal for this project was to start getting more comfortable with Javascript by manipulating arrays, objects and DOM elements as well as understanding the logic of the game and the steps that we needed to follow in order to make it work. 
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I approached the problem by carefully reading the instructions and trying many times to find the correct solution in each case. I also used pen and paper to completely wrap my head around the concepts that we were supposed to digest and asked many questions on the live sessions to be completely sure about the sintaxis of the elements that were already in the code when I started. It wasn't an easy process and my work is a result of imitating the steps that I saw in the examples, but now I feel more confident about the code flow and the correct way to approach this kind of problem. if I had more time (and experience on how to do it) I would like to change the cards to use other characters and add sound effects and score board.
 
-## View it live
+<h2>View it live</h2>
 
-Have you deployed your project somewhere? Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+You can see and play the game online by following this link: https://app.netlify.com/sites/estefania-project5-guesswhogame/overview
```

Comment:
This link seem to require that you are logged in to Netlify, so I believe it might be your personal dashboard view. If instead using https://estefania-project5-guesswhogame.netlify.app/ it will lead directly to the published project without login.

Reply:
@esteficodes please change this one 😻